apps/02RockPaperScissors.js

```diff
@@ -10,37 +10,64 @@ var rl = readline.createInterface({
 
 function rockPaperScissors(hand1, hand2) {
 
-  // Write code here
+  if (hand1 === hand2) {
+    return "It's a tie!"
+  } else if (hand1 === "rock") {
 
+    if (hand2 === "scissors") {
+      return "Hand one wins!";
+    } else {
+      return "Hand two wins!";
+    }
+
+  } else if (hand1 === "paper") {
```

Comment:
I feel like this is the best syntax, but I always want to put my "else if" on separate lines like the following...

if () {
}

else if () {
}

Just because it's easier to read!